tool/tctl/common/tctl.go

```diff
@@ -165,9 +165,13 @@ func Run(commands []CLICommand) {
 
 	client, err := authclient.Connect(ctx, clientConfig)
 	if err != nil {
+		addr := "UNKNOWN"
+		if len(cfg.AuthServers) > 0 {
+			addr = cfg.AuthServers[0].Addr
+		}
 		utils.Consolef(os.Stderr, log.WithField(trace.Component, teleport.ComponentClient), teleport.ComponentClient,
 			"Cannot connect to the auth server: %v.\nIs the auth server running on %q?",
-			err, cfg.AuthServers[0].Addr)
+			err, addr)
```

Comment:
Unless I misunderstood something, this should work (and gets rid of the weird `"UNKNOWN"` thing).

```suggestion
			err, clientConfig.AuthServers[0].Addr)
```